### PR TITLE
Implemented SortBy Dropdown selector.

### PR DESCRIPTION
**Pull request by IlanSirisky**
Implemented SortBy Dropdown selector using MUI Select.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+// import TestTypography from "./components/testComponents/TestTypography";
+
+const options = [
+  { value: "az", label: "Alphabetical A-Z" },
+  { value: "za", label: "Alphabetical Z-A" },
+  { value: "powerHighToLow", label: "Power (High to low)" },
+  { value: "powerLowToHigh", label: "Power (Low to high)" },
+  { value: "hpHighToLow", label: "HP (High to low)" },
+  { value: "hpLowToHigh", label: "HP (Low to high)" },
+];
```
Comment (YuvalLeviMoveoHLS) [inline src/App.tsx]: 1. move to consts file
2. create interface

File: src/components/SortBySelector/SortBySelector.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import SortIcon from "../../assets/icons/SortIcon.svg";
+import ActiveSortIcon from "../../assets/icons/ActiveSortIcon.svg";
+
+interface SortBySelectProps {
+  options: { value: string; label: string }[];
+  placeholder?: string;
+  sx?: object;
+}
+
+const SortBySelect = ({
```
Comment (ZivCohenMoveo) [inline src/components/SortBySelector/SortBySelector.tsx]: Change all the naming to be generic select/dropdown(without sortBy)

File: src/components/SortBySelector/SortBySelector.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  options,
+  placeholder = "Sort by",
```
Comment (ZivCohenMoveo) [inline src/components/SortBySelector/SortBySelector.tsx]: remove "sort by"

File: src/components/SortBySelector/SortBySelector.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  sx = {},
+}: SortBySelectProps) => {
+  const [sortBy, setSortBy] = useState<string>("");
```
Comment (ZivCohenMoveo) [inline src/components/SortBySelector/SortBySelector.tsx]: naming

File: src/components/SortBySelector/SortBySelector.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  // Manage focus state to render the correct icon
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => {
+    setIsFocused(false);
+  };
+
+  return (
+    <StyledFormControl variant="outlined" sx={sx}>
+      {!sortBy && (
+        <StyledInputLabel shrink={!!sortBy} htmlFor="sort-by-select">
```
Comment (ZivCohenMoveo) [inline src/components/SortBySelector/SortBySelector.tsx]: shrink will never be "true" due to the condition `!sortBy &&` in line 38. maybe can be `shrink={false}`

**Review by ZivCohenMoveo — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (IlanSirisky) [reply src/App.tsx]: done.

File: src/components/SortBySelector/SortBySelector.tsx (hunk 2 above)
Comment (IlanSirisky) [reply src/components/SortBySelector/SortBySelector.tsx]: Updated all names to be Select

File: src/components/SortBySelector/SortBySelector.tsx (hunk 5 above)
Comment (IlanSirisky) [reply src/components/SortBySelector/SortBySelector.tsx]: Updated to be ```shrink={false}```

**Review by YuvalLeviMoveoHLS — APPROVED** [review]

**Pull request merged**